SEM/datamodels/plugins/hitachi.py

```diff
@@ -0,0 +1,81 @@
+from pathlib import Path
+import dlite
+
+import configparser
+from PIL import Image
+from PIL.TiffTags import TAGS
+
+
+def read_ini(fname : str):
```

Comment:
Since DLite plugins are parsed in a common scope by the embedded interpreter, the plugins should only contain the `DLiteStorageBase` subclass to avoid naming conflicts between plugins.

Hence, make `read_ini()` a `@staticmethod in the `HitachiStorage` class.